### DISCUSSION TITLE
Use of discussions?

### DISCUSSION THREAD

**Discussion by patrickhlauke** [w3c/wcag#1776] [General]
While I can understand the desire to keep "Issues" as clean actual issues, rather than more general discussions/questions ... are notifications (by email) sent when a new thread/discussion is opened? I don't recall seeing any when @mraccess77 posted the existing two ones here...

Is there a danger of spreading discussions even more thinly now between mailing lists (IG and GL), issues, PRs, and now discussions?

**Comment by detlevhfischer** [comment]
Yes, that's one reason I prefer putting things into issues - too many places.

**Comment by bruce-usab** [comment]
> are notifications (by email) sent when a new thread/discussion is opened?

Does not seem to be the case.  I do see "Notifications:  You’re receiving notifications because you’re watching this repository."  But that might now only be because I just commented on @mraccess77 other discussion?

**Reply by bruce-usab** [reply]
Quite recently (8/25/22022) I got a new discussion notice to my email, similar to Issues, PR etc.  I haven't changed any GitHub settings, but it was a welcomed surprise.

**Comment by yatil** [comment]
I guess you’re subscribed automatically when one comments, but not automatically notified only because you watch the repository. I think for detailed discussions, this might be the better place and then only bring decisions into actual “issues”. I like that you can reference discussions like PRs and Issues by number.

**Comment by alastc** [comment]
This is new functionality from github, not specific to us!

We need to get a feel for the features, e.g. this appears to have voting or ordering like stackoverflow?

I can't see any notification features though, so I assume you 'subscribe' to a discussion based on interacting with it.

We might start moving issues into discussions if they don't have an obvious update to make to the docs.

**Reply by patrickhlauke** [reply]
but it appears you currently then have to continue manually looking at the discussions page to check if any new discussions were posted. so compared to mailing lists and github notifications for new issues/PRs that you can opt in and then receive, for discussions you now have to explicitly go check here. inefficient, in my view...

**Reply by yatil** [reply]
@patrickhlauke You can watch discussions, too, if you want. There is a dropdown on the top right: “Watch”.

If you select “Custom”, you are able to select “Discussions”. If I understand that correctly, it would inform you about all discussions.

![Screenshot of dropdown](https://user-images.githubusercontent.com/58862/116973020-2fc1ef00-acbc-11eb-9d76-e7c0f9b996da.png)

I personally prefer to be not notified about all messages in this repository and instead have only ”Participating and @mentioned” selected.

**Reply by patrickhlauke** [reply]
aha, so likely that most folks (like myself) had this already set to just "issues" or similar, and as a result didn't get notifications when @mraccess77 started using the discussion feature...which is why his two discussions ended up into the void

the fact that i can at least now explicitly enable notifications for new discussions (presumably) softens my take slightly, but still feels like this makes it yet another place for decisions/conversations. and a priori it's not always clear if something is an issue, or a discussion ... not quite clear-cut from the outset.

**Comment by bruce-usab** [comment]
> We might start moving issues into discussions if they don't have an obvious update to make to the docs.

How about asking folks to start with Discussions?  It feel like Issues is a fair enough venue to ask about SC &mdash; but it is a minority which results in changes to SC or Understanding regardless of the useful and expert information on display.  Those issues thread just get close, without memorizing the highlights.
